lib/jnpr/junos/device.py

```diff
@@ -1145,6 +1155,10 @@ def __init__(self, *vargs, **kvargs):
             default is ``False`` to use DOM.
             Select ``True`` to use SAX (if SAX input is provided).
 
+        :param int conn_open_timeout:
+            *OPTIONAL* To specify the timeout in seconds, which will
+            be used while opening SSH connection to the device
```

Comment:
but above you said, this will be used for console connection too..

Reply:
Should I remove `SSH` or add `console` to the doc ??